python/lsst/daf/butler/queries/tree/_column_literal.py

```diff
@@ -374,6 +383,12 @@ def make_column_literal(value: LiteralValue) -> ColumnLiteral:
             return DateTimeColumnLiteral.from_value(astropy.time.Time(value))
         case Timespan():
             return TimespanColumnLiteral.from_value(value)
-        case Region():
+        case lsst.sphgeom.Region():
             return RegionColumnLiteral.from_value(value)
+        case lsst.sphgeom.LonLat():
+            vec = lsst.sphgeom.UnitVector3d(value)
+            # Convert the point to a Region by representing it as a zero-radius
+            # Circle.
```

Comment:
I wasn't sure whether zero-radius would work, but if it does, great!

Reply:
It seems to, and from the docs it seems to be intended to with slight caveats (e.g. they go into a discussion of an edge case where two zero-radius circles that are very close but not exact might still be considered overlapping.)